docs/steering/meeting-notes/2025-12-01.md

```diff
@@ -12,26 +12,37 @@
 
 ### Technical Steering Committee Representatives
 
-- [ ] Jakob Möller - SAP
+- [X] Jakob Möller - SAP
 - [ ] Gerald Morrison - SAP
-- [ ] Fabian Burth - SAP
+- [X] Fabian Burth - SAP
 - [ ] Frederic Wilhelm - SAP
-- [ ] Philipp Heil - SAP
+- [X] Philipp Heil - SAP
 - [ ] Gergely Bräutigam - Individual Contributor
 
-Quorum: ✅ / ❌
-
-### Non-Voting Representatives or Observing Participants
-
-- [ ] Member
+Quorum: ❌
 
 ## Agenda Items
 
-- [ ] Revisit Action Items
-- [ ] Update on TAC / TSC
-    - [ ] Update on the NeoNephos Guidelines
-    - [ ] Update on the OCM Spec License
-- [ ] Confirm SIG Runtime Chair / Tech Lead Nomination
+- [X] Jakob to talk to Florian Müller (TAC Chair on NeoNephos Foundation) to escalate Legal Discussion / Decision on License of OCM Spec.
+    - [X] Jakob to update on License Compliance regarding OCM Specification
+      - Discussion was escalated and brought into the NeoNephos Governing Board. No update yet.
+      - No official approval, but tacit acceptance of the current situation.
+      - Technically migration to NeoNephos Github Org is possible now, but missing Billing Information setup.
+- [X] Jakob will talk with Mirza to get all OCM Project Members become part of NeoNephos so that TAC issues become visible.
+  - Jakob thinks he can promote all OCM Projects Members to NeoNephos Org. Previously there was a seat limit, now no longer the case.
+  - Only reason OCM team is not invited yet, is because Mirza and Jakob want to prevent a flood of invitations.
+  - Jakob to invite all OCM Project Members to NeoNephos Org when Automation is in place.
+  - Philipp: Maybe we can automate with https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/
+    - Jakob to check out if we can work standalone
+    - Jakob is currently also looking at safe-settings: https://github.com/github/safe-settings
+  - Migration Prototype is happening already with platform-mesh project, and for the time being we will likely setup SAP billing as replacement for LF billing.
+- [ ] Pending Escalation of Jakob to confront NeoNephos Project Governing Board on why TAC member is not allowed to contribute to NeoNephos Project Lifecycle completion.
+  - Jakob didn't manage to push out this escalation item and will push this out to next meeting.
+- [ ] Gerald to update the SIG Charter - Postponed, Gerald not present. (likely updated already)
+- [ ] Jakob to announce SIG Runtime Contribution Possibility to the NeoNephos Project
+  - This will happen as a general announcement to start 2026. 
+  - Contribution into the SIG is likely powered solely by existing members of OCM, and externals will come slowly based on NeoNephos impact
+  - First candidates are StackIT and other NeoNephos members already showing interest in OCM.
```

Comment:
This is a lot of things for you Jakob. Is there ANY way we or the other TSC members could help you?

Reply:
the biggest issue is that youre not allowed to be put into the TAC right now. I think maybe we could setup a Co-Chair with which we could split work. WDYT?